src/main/java/frc/robot/swerve/SwerveConfig.java

```diff
@@ -58,48 +49,32 @@ public class SwerveConfig {
     public static final int BR_TURN_CANID = 20;
     public static final boolean BR_REVERSE = false;
 
-    // helper function
-
-
-
-
-
-
-    ///////////////////////////////////////////// UNDERCONSTRUCTION
-
-
-
-
-    //The createDriveMotor method 
-
-    public static void createDriveMotor(int canld, boolean reverseMotor){
```

Comment:
Your method creates the motor, but then doesn't return it. The caller of the method won't be able to use the motor you create. I updated this by:
* Adding a return type to the method (not `void`); and,
* Adding a return statement at the end (`return driveMotor`)